server/metrics/metrics.go

```diff
@@ -351,7 +365,7 @@ var (
 		Namespace: bbNamespace,
 		Subsystem: "remote_cache",
 		Name:      "disk_cache_seconds_since_last_access",
-		Buckets:   prometheus.ExponentialBuckets(1, 10, 9),
+		Buckets:   durationSecBuckets,
```

Comment:
@tylerwilliams I missed this in the last PR, but it looks like the existing metrics all use usec. Mind if I change this metric + impl to usec as well?

Reply:
Ah yeah I changed it in that last CL because the buckets were reasonable for seconds but not for usec and I figured I only needed second granularity for this metric.

But yeah, OK w/ me if you want to go back to usec and change the buckets too.